### PR TITLE
Update `CustomProperties` type to `array`

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by chenrui333** [comment]
When trying to generate the client code from the `openapi.yaml`, I have encountered the type issue:

```
The swagger spec at "/go/src/github.com/mlafeldt/go-launchdarkly/swagger.yaml" is invalid against swagger specification 2.0. see errors :
- definitions.CustomProperties in body must be of type array

make: *** [generate] Error 1
```

Correcting the type to `array` and fixed the code generation problem.

**Comment by chenrui333** [comment]
circleci build failure due to the submodule access issue:
```
Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'https://github.com/launchdarkly/api-client-bash' into submodule path '/home/circleci/project/targets/api-client-bash' failed
Makefile:93: recipe for target 'load_prior_targets' failed
make: *** [load_prior_targets] Error 128
Exited with code 2
```

File: spec/definitions.yaml [hunk 1]
```diff
@@ -732,7 +732,7 @@ PatchOperation:
     - path
     - value
 CustomProperties:
-  type: object
+  type: array
```
Comment (chenrui333) [inline spec/definitions.yaml:735]: relates to https://github.com/mlafeldt/go-launchdarkly/pull/2/files#diff-bac86ae86657c2fa2c3dd2ab26df4960R1610

**Comment by ashanbrown** [comment]
@chenrui333 Thanks for bringing this issue to our attention.  We believe that the CustomProperties should be an object (a map) rather than an array, but it looks like it wasn't described correctly.  I've opened a PR at https://github.com/launchdarkly/ld-openapi/pull/14 to fix this.  Please let me know if this might fix your problem.  We will merge this change shortly.

**Comment by chenrui333** [comment]
Sure, I can certainly test your change.

**Comment by chenrui333** [comment]
Close in favor of #14

**Comment by chenrui333** [comment]
@ashanbrown Yeah, just tested your branch, it works for me.

**Pull request closed without merging**